enterprise/migrations/0149_invite_key_required_default_expiry_backfill.py

```diff
@@ -0,0 +1,27 @@
+from django.db import migrations
+from dateutil.relativedelta import relativedelta
+    
+
+def backfill_null_expiry_date_for_invite_keys(apps, schema_editor):
+    """
+    Finds all EnterpriseCustomerInviteKey objects with null expiration_date and applies a default
+    expiration_date of 1-year from when the EnterpriseCustomerInviteKey object was created as a
+    reasonable fallback such that there are no objects with expiration_date as null so a future
+    migration can require this field at the Django admin and database levels.
+    """
+    EnterpriseCustomerInviteKey = apps.get_model('enterprise', 'EnterpriseCustomerInviteKey')
+    queryset = EnterpriseCustomerInviteKey.objects.filter(expiration_date__isnull=True)
+    for invite_key in queryset:
+        invite_key.expiration_date = invite_key.created + relativedelta(years=1)
```

Comment:
I've never seen a `relativedelta` before!  Probably equivalent to a `timedelta` in this case, right?

Reply:
It is essentially equivalent to `timedelta`. I'm not sure why I went with `relativedelta` here, tbh. I've changed it back to `timedelta` instead.